typescript/eslint.config.js

```diff
@@ -0,0 +1,4 @@
+import prettier from "eslint-config-prettier";
+import svelte from "eslint-plugin-svelte";
```

Comment:
Found this change a bit confusing as I don't see this listed in the package.json dependencies, does this work as is? Can you explain? Or does it only work because you have it globally installed on your machine?